docs/components/QualityChecklist.js

```diff
@@ -0,0 +1,71 @@
+// @flow strict
+import { type Node } from 'react';
+import { Box, Text, Table, SlimBanner } from 'gestalt';
+import COMPONENT_DATA from './COMPONENT_DATA.js';
+import MainSection from './MainSection.js';
+import StatusData from './StatusData.js';
+import { STATUS_DESCRIPTION, COMPONENT_STATUS_MESSAGING } from './COMPONENT_STATUS_MESSAGING.js';
+
+type Props = {|
+  component: string,
+|};
+
+export default function QualityChecklist({ component }: Props): Node {
+  const componentData = [
+    ...COMPONENT_DATA.buildingBlockComponents,
+    ...COMPONENT_DATA.generalComponents,
+    ...COMPONENT_DATA.utilityComponents,
+  ].find((cmpName) => cmpName.name === component);
+
+  return (
+    <MainSection name="Component Quality Checklist">
```

Comment:
I think this should be "Component quality checklist"? Isn't that our preferred style?

Reply:
DONE